pnas/PNAS-template-main.tex

```diff
@@ -230,44 +231,45 @@ \subsection*{Stability of defection}\label{subsection:stability_defection}
 population all the members will over time adopt the same behaviour; thus in such
 population cooperation will never take over. This is demonstrated in
 Fig.~\ref{fig:stable_defection} and Fig.~\ref{fig:unstable_defection}.
+These have been simulated using~\cite{axelrodproject} an open
+source research framework for the study of the IPD.
 
 \begin{figure}
+    \begin{subfigure}[t]{0.5\textwidth}
         \centering
         \includegraphics[width=.8\linewidth]{population_defection_takes_over.pdf}
-        \caption{For opponents \(q_{1}=(\frac{371}{1250},\frac{4693}{25000},\frac{4037}{50000},\frac{18461}{25000})\),
-        $q_{2}=(\frac{48841}{100000},\frac{30587}{50000},\frac{76591}{100000},\frac{25921}{50000})$ and
-        $q_{3}=(\frac{22199}{100000},\frac{87073}{100000},\frac{646}{3125},\frac{91861}{100000})$
-        conditions (\ref{eq:defection_condition_one}) and
-        (\ref{eq:defection_condition_two}) hold and Defector takes over the
-        population.}
-        \label{fig:stable_defection}
-\end{figure}
-\begin{figure}
+        \caption{\(q_{1}=(0.22199, 0.87073, 0.20672, 0.91861)\),
+        $q_{2}=(0.48841, 0.61174, 0.76591, 0.51842)$ and
+        $q_{3}=(0.2968, 0.18772, 0.08074, 0.73844)$}\label{fig:stable_defection}
+    \end{subfigure}
+    \begin{subfigure}[t]{0.5\textwidth}
         \centering
         \includegraphics[width=.8\linewidth]{population_defection_fails.pdf}
-        \caption{For opponents $q_{1}=(\frac{69773}{100000},\frac{21609}{100000},\frac{97627}{100000},\frac{623}{100000})$,
-        $q_{2}=(\frac{12649}{50000},\frac{43479}{100000},\frac{38969}{50000},\frac{19769}{100000})$ and
-        $q_{3}=(\frac{96703}{100000},\frac{54723}{100000},\frac{24317}{25000},\frac{35741}{50000})$
-        (\ref{eq:defection_condition_one}) fails and
-        (\ref{eq:defection_condition_two}) holds and Defector does not take over
-        the population.}
-        \label{fig:unstable_defection}
+        \caption{$q_{1}=(0.96703, 0.54723, 0.97268, 0.71482)$,
+        $q_{2}=(0.69773, 0.21609, 0.97627, 0.0062)$ and
+        $q_{3}=(0.25298, 0.43479, 0.77938, 0.19769)$}\label{fig:unstable_defection}
+    \end{subfigure}
+    \caption{Fig.~\ref{fig:stable_defection} Eq.~\ref{eq:defection_condition_one} and
+    Eq.~\ref{eq:defection_condition_two} hold and Defector takes over the
+    population. Fig.~\ref{fig:unstable_defection} Eq.~\ref{eq:defection_condition_one} fails and
+    Eq.~\ref{eq:defection_condition_two} holds and Defector does not take over
```

Comment:
fails ~~and Eq.~\ref{eq:defection_condition_two} holds~~ 

(I think, let's chat to be sure)